README.md

```diff
@@ -1,2 +1,141 @@
 # mx-chain-ws-connector-template-go
-Template for WebSocket connector written in go
+
+## Introduction
+
+In today's rapidly evolving blockchain ecosystem, the demand for real-time access to exported data from blockchain nodes
+has significantly increased. To meet this growing need within our expanding system, we have developed the Outport
+Driver, a powerful websocket connection system specifically designed for the Multiversx blockchain.
+
+The Outport Driver acts as a crucial bridge between the Multiversx blockchain node and various services that rely on
+receiving up-to-date data from the node. By establishing a websocket connection, the Outport Driver enables seamless
+transmission of essential information, including blocks, validator details, and much more.
```

Comment:
I would remove "much more". Each time I see this I'm thinking of "french fries". I do not think this websocket connector can deliver "french fries" :D

```
... Outport Driver enables seamless
transmission of essential information, including blocks, validator details, account changes, and other processing results.
```